internal/command/command.go

```diff
@@ -39,7 +40,8 @@ func (c Command) toCmd(ctx context.Context, config *Config, cmdArgs *Args, outpu
 
 	for i, a := range rawArgs {
 		tpl, err := template.New(c.ID).Option("missingkey=error").Funcs(template.FuncMap{
-			"trim": strings.TrimSpace,
+			"trim":    strings.TrimSpace,
+			"jsonGet": gjson.Get,
```

Comment:
Why is it get? Would there ever be a need for any other operation?

Reply:
In other words shouldn't this be `json` because getting is the only thing a template does

Reply:
ah sure, it was get because thats what the function was called, but you make a good point!